### DISCUSSION TITLE
邀请人学习move赚SUI记录

### DISCUSSION THREAD

**Discussion by uvd** [hoh-zone/SuiStartrek#80] [General]
# 奖励 
每邀请一个人完成第一阶段的学习可获得 10SUI的奖励
# 记录方式  每个人一个评论卡  这是最后进群的记录
<img width="255" alt="image" src="https://github.com/movefuns/SuiStartrek/assets/10650592/554e21d0-d8c4-4639-906a-42a41f1921f1">

- [ ] githubid  ，微信id, 邀请日期

例如 

- [ ] zhangsan, 小不点 ，11月20号入群
- [ ] zhaosi, 大的 ，11月20号入群

**Comment by uvd** [comment]
# uvd 的邀请记录
- [ ] zhangsan, 小不点 ，11月20号入群
- [ ] zhaosi, 大的 ，11月20号入群

**Comment by jeffierw** [comment]
# jeffierw的邀请记录

- [x] futuwxq，xiaoQ3024881272， 11月10号入群
